ownCloud/Static Login/Interface/StaticLoginStepViewController.swift

```diff
@@ -132,11 +132,35 @@ extension StaticTableViewSection {
 	}
 
 	@discardableResult
-	func addButtonFooter(proceedLabel: String? = nil, cancelLabel : String? = nil, topSpacing : CGFloat = 30) -> (UIButton?, UIButton?) {
+	func addButtonFooter(message: String? = nil, messageItemStyle: ThemeItemStyle = .title, proceedLabel: String? = nil, proceedItemStyle: ThemeItemStyle = .approval, cancelLabel : String? = nil, topSpacing : CGFloat = 30) -> (UIButton?, UIButton?) {
 		let containerView = FullWidthHeaderView()
 		var continueButton : ThemeButton?
 		var cancelButton : UIButton?
 		var constraints : [NSLayoutConstraint] = []
+		var topAnchor = containerView.topAnchor
+
+		if message != nil {
+			let titleLabel = UILabel()
+			titleLabel.translatesAutoresizingMaskIntoConstraints = false
+			titleLabel.setContentHuggingPriority(.defaultLow, for: .horizontal)
+
+			titleLabel.text = message
+			titleLabel.textAlignment = .center
+			titleLabel.numberOfLines = 0
+			containerView.addSubview(titleLabel)
+
+			containerView.addThemeApplier({ (_, collection, _) in
+				titleLabel.applyThemeCollection(collection, itemStyle: messageItemStyle)
+			})
+
+			NSLayoutConstraint.activate([
+				titleLabel.leftAnchor.constraint(equalTo: containerView.leftAnchor),
+				titleLabel.rightAnchor.constraint(equalTo: containerView.rightAnchor),
+				titleLabel.centerXAnchor.constraint(equalTo: containerView.centerXAnchor),
```

Comment:
The X position is already determined by

```swift
titleLabel.leftAnchor.constraint(equalTo: containerView.leftAnchor),
titleLabel.rightAnchor.constraint(equalTo: containerView.rightAnchor),
```

so that you could remove

```swift
titleLabel.centerXAnchor.constraint(equalTo: containerView.centerXAnchor),
```

Reply:
fixed